scripts/in_container/run_prepare_er_diagram.py

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env python3
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+"""
+Module to update db migration information in Airflow
+"""
+import os
+from pathlib import Path
+
+AIRFLOW_SOURCES_ROOT = Path(__file__).parents[2].resolve()
+PNG_FILE = AIRFLOW_SOURCES_ROOT / "docs" / "apache-airflow" / 'img' / 'airflow_erd.png'
+if __name__ == '__main__':
+    from eralchemy2 import render_er
+
+    render_er(os.environ.get("AIRFLOW__DATABASE__SQL_ALCHEMY_CONN"), os.fspath(PNG_FILE))
```

Comment:
Can we make this a SVG rather than a PNG? (dot supports this).

Or another option: can we have this generate it as a graphviz .dot file? That way we don't have the png checked in, just the `.dot` file which sphinx can render with `:: graphviz: file.dot`

Reply:
Yeah. I think now we can do it - because I actually added graphviz as a dependency to our CI image (needed by eralchemy), so we could do that (providing that most people already use `breeze build-docs` to build image which I think they do - otherwize it would put an extra requirement to have graphviz installed, which would likely make it a bit less "friendly" for the casual users (graphviz is a system level dependency). It would also undermine our efforts to get it working on windows (but lucklly bulding docs inside the CI image of ours solves the problem).

Reply:
BTW, Graphviz dependency is the main reason why the pre-commit for that is two-step and uses CI image :).

Reply:
BTW. Using Docker image for our common environment was specifically chosen to handle those kind of problems.